schemas/components/schemas/VPNMethods.yaml

```diff
@@ -20,3 +21,7 @@ properties:
     type: boolean
     description: >-
       The browser runs on a different operating system than the operating system inferred from the request network signature.
+  relay:
```

Comment:
@soniabhishek Can you [generate a changeset for the PR](https://github.com/fingerprintjs/fingerprint-pro-server-api-openapi/blob/main/CONTRIBUTING.md#describing-changes) please?

Reply:
I gave this a try @JuroUhlar , please check my last commit

Reply:
Just help me select the proper `scope` for this change

Reply:
Thank you, let's discuss in this thread: https://github.com/fingerprintjs/fingerprint-pro-server-api-openapi/pull/174/files#r1850249236